components/common/ShareButtons.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faFacebook, faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { trackEvent } from 'utils';
 
-const ShareButtons = ({ className }) => (
+const ShareButtons = ({ className, context }) => (
```

Comment:
We use the `context` prop to determine if this is being rendered on the home page or state page, and then use that to record a click on either of the buttons.